Kudu.Services.Web/Startup.cs

```diff
@@ -83,6 +83,16 @@ public Startup(IConfiguration configuration, IWebHostEnvironment hostingEnvironm
         /// </todo>
         public void ConfigureServices(IServiceCollection services)
         {
+            if (FileSystemHelpers.DirectoryExists("/home/apps"))
```

Comment:
what if the pod crash when moving the folder, please have a test

Reply:
have tested it. When moving Directory operation interrupted and re-execute  the moving Directory operation, everything seems fine.